tests/components/selection_input.test.ts

```diff
@@ -263,6 +263,23 @@ describe("Selection Input", () => {
     app.destroy();
   });
 
+  test("Can use arrow keys to change ranges after input corruption", async () => {
```

Comment:
Since you are indirectly fixing the "handleEvent" function of the plugin  (unless I'm mistaken), the test should take place in the selection input plugin  ^^ no need to instanciate a whole component.